news/2017-10-12-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "The QAT Gazette: October 2017"
+date: 2017-10-12 09:00:00 +0000
+---
+
+There have been major developments in the Quality Assurance Team and Beatmap Nominators this month, with new QAT leaders, changes to the Nominator Tier system, revival of mapping contests, and more ranked maps per day! Curious to know more? Read on to find out!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+Hello again everyone! Another month has passed, so here we are with the second issue of the **QAT Gazette**!
+New Beatmap Nominators have joined for osu!taiko, osu!catch and osu!mania, the QAT and Beatmap Nominator Upheaval is underway and Mapping with Rewards has made a return - these are just some of the topics in this month’s issue, so let’s begin!
+
+## The Upheaval
+
+The Upheaval was probably the biggest event of changes affecting the Quality Assurance Team and the Beatmap Nominators we have seen for a long time. For those of you who may have missed the details, you can still read more about it in the [Development Forum Thread](https://osu.ppy.sh/community/forums/topics/635507), but the main outcomes are explained below!
+
+### No More Tiers
+
+As we discussed last month, the biggest point in the Upheaval was **the removal of the Beatmap Nominator Tier system**. Previously, we split the osu! Beatmap Nominators into two groups - Tier 1 and Tier 2. Tier 1 members were able to bubble maps only, whereas Tier 2 members were able to bubble and also qualify beatmaps. 
+
+Many people were not happy about these changes because of the lack of Tier 2 Nominators. The amount of bubbled beatmaps increased, while the amount of Qualified beatmaps was reduced a lot. With the Upheaval, we have replaced the Tier system with an alternative **“probation” system** (more of this later in the blog!), allowing all full member Beatmap Nominators to both bubble and qualify beatmaps again! 
+
+### Beatmap Nominator Rewards
+
+You may have already seen that all Beatmap Nominators received a “Beatmap Nominator” forum title and a special purple forum color. The older players among you will maybe know this color, as it’s really similar to the old Mapping Assistance Team colour. It is planned for members of the Beatmap Nominators to also receive further rewards for their effort, including special titles for consistently active members over long periods of time. How we judge the awarding of this **Elite Nominator** title is still being decided on.
```

Comment:
"The older players among you will maybe know this **_color,_**"

color :angery: